content/en/events/_index.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Events
+linkTitle: Events
+toc_hide: false
+list_pages: false
+menu:
+  main:
+    weight: 20
+description: Events related to Cloud Native Finland
+---
+
+## Kubernetes Comminity Day Helsinki 2025
+
+We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community (us) organized event
```

Comment:
```suggestion
We will have a [KCD event](https://www.cncf.io/kcds/) in Helsinki on May 6, 2025! This community-organized event
```

Reply:
Do we need the hyphen here?

Reply:
even if not the hyphen, the (us) in between made the reading hard. So, maybe just `community organized event`

Reply:
Fixed in [2dbfe34](https://github.com/cloud-native-finland/cloud-native-finland-site/pull/2/commits/2dbfe340356fd982ff6fe40cf504dd51a975eb8f)